docs/docs/data-annotation.md

```diff
@@ -112,15 +112,19 @@ with:
     - Due Date: `🗓️YYYY-MM-DD`
     - Completed Date: `✅YYYY-MM-DD`
     - Created Date: `➕YYYY-MM-DD`
+    - Start Date: `🛫YYYY-MM-DD`
+    - Scheduled Date: `⏳YYYY-MM-DD`
 === "Example"
-    - [ ] Do this saturday 🗓️2021-08-29.
+    - [ ] Due this saturday 🗓️2021-08-29.
```

Comment:
It's interesting to me that all these examples have full-stops at the end of the lines: `.`

As far as I know, this will mean that the Tasks plugin won't (currently) recognise the emoji earlier in the line, as it reads from the right until it finds something it does not recognise. (As of 1.9.0, it recognises the signifiers with dates, and tags when parsing task lines(